ckan/setup/GSA_prerun.py

```diff
@@ -14,6 +16,25 @@
 RETRY = 5
 
 
+def init_db():
+
+    db_command = ["ckan", "-c", pr.ckan_ini, "db", "init"]
+    print("[prerun] Initializing or upgrading db - start")
+    try:
+        subprocess.check_output(db_command, stderr=subprocess.STDOUT)
+        print("[prerun] Initializing or upgrading db - end")
+    except subprocess.CalledProcessError as e:
+        # GSA FIX: e.output is bytes upstream
+        if "OperationalError" in str(e.output):
```

Comment:
Can we push this fix upstream?

Reply:
I'm not sure.  I think this is a PY2 vs PY3 issue that upstream is still allowing support for PY2 😕